assetsTransformer.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
```

Comment:
⚠️ Note: This and file below are also present in Steven's review, which means that they will create a merge conflict for each other.